### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by meghalithic**
removing things that aren't needed and fixing links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,27 +7,27 @@ The image segmentation workflow is managed using snakemake, a user-friendly pyth
 
 The segmentation workflow consists of the following steps each defined by a "rule". 
 The output of each rule is store to specific folder that is produced by the worrkflow:
```
Comment (johnbradley) [inline README.md]:
```suggestion
The output of each rule is stored to a specific folder that is produced by the workflow:
```

File: README.md [hunk 2]
```diff
@@ -7,27 +7,27 @@ The image segmentation workflow is managed using snakemake, a user-friendly pyth
...
-   1. Download the fish **Images** from [Tulane server](http://www.tubri.org/HDR/INHS/) using a simple bash script.
+   1. Download the fish **Images** from [Fish-AIR](https://fishair.org/) using a simple bash script.
       * Images are saved to the folder **Images**.
```
Comment (johnbradley) [inline README.md]: Your changes are fine, but the text is misleading. The step just downloads images based on URLs in the CSV file. They could come from anywhere. The CSV needs to have a `path` that is actually a URL and a `original_file_name` that will have the filename extension removed and that will be the unique identifier for the image. We also don't use a bash script.
```suggestion
   1. Download the fish **Images** from URLs within the input CSV file. The CSV file must have a column named `path` that contains the URL of the image. The CSV file must have a column named `original_file_name` that will be used to create a unique filename for the images downloaded.
      * Images are saved to the folder **Images**.
```

File: README.md [hunk 3]
```diff
@@ -7,27 +7,27 @@ The image segmentation workflow is managed using snakemake, a user-friendly pyth
...
    
-   2. Extract **Metadata** information using Detectron2 (deep learning segmentation). The code developed by Drexel and the script used can be found [here](https://github.com/hdr-bgnn/drexel_metadata/blob/Thibault/gen_metadata_mini/scripts/gen_metadata.py).
+   2. Extract **Metadata** information using Detectron2 (deep learning segmentation). The code developed by Drexel and the script used can be found [here](https://github.com/hdr-bgnn/drexel_metadata).
       * This code extracts the ruler scale (pixels/cm).
       * This code identified and extracts a bounding box around the fish in the image.
       * Information about the ruler scale and bounding box size are saved as .json files in the folder **Metadata**.
-      * Images retain the original file name and we add the suffix "_mask".
+      * Images retain the file ARKID and we add the suffix "_mask".
```
Comment (johnbradley) [inline README.md]:
```suggestion
      * The output filename is based on the image filename adding the filename suffix "_mask" and changing the extension.
```

File: README.md [hunk 4]
```diff
@@ -7,27 +7,27 @@ The image segmentation workflow is managed using snakemake, a user-friendly pyth
...
       * The mask is saved in the folder **Mask**.
       
    3.	Create **Cropped** images of the fish using the bounding box from Metadata. The code is under [Crop_image_main.py](https://github.com/hdr-bgnn/Crop_image/blob/main/Crop_image_main.py).
       * We increased the bounding box around the fish by 2.5% on each side (5% total) for the crop to prevent truncation of the file.
-      * Images retain the original file name and we add the suffix "_cropped".
+      * Images retain the file ARKID and we add the suffix "_cropped".
```
Comment (johnbradley) [inline README.md]:
```suggestion
      * The output filename is based on the image filename adding the filename suffix "_cropped".
```

File: README.md [hunk 5]
```diff
@@ -7,27 +7,27 @@ The image segmentation workflow is managed using snakemake, a user-friendly pyth
...
       * The cropped image is in the folder **Cropped**.
       
    4. **Segmented** traits using code developed by Maruf and reorganized by Thibault [here](https://github.com/hdr-bgnn/BGNN-trait-segmentation/blob/main/Segment_mini/scripts/segmentation_main.py).
       * This code produces an image of a segmented fish (color coding following example in "Stage 1" below).
       * The input cropped image must be resized to 800x320 pixels.
       * We then resize the output segmented image (800x320) to the size of the cropped image (which is the size of the bounding box plus 5% increase).
            - This ensures that the image is at the same scale as the ruler when the ruler scale was extracted in Metadata.
-      * Images retain the original file name and we add the suffix "_segmented".
+      * Images retain the file ARKID and we add the suffix "_segmented".
```
Comment (johnbradley) [inline README.md]:
```suggestion
      * The output filename is based on the image filename adding the filename suffix "_segmented".
```

File: README.md (hunk 1 above)
Comment (hlapp) [reply README.md]: @johnbradley I added two more small corrections. Can you double check?

**Comment by meghalithic** [comment]
Thanks for the help @johnbradley and @hlapp, I like the changes you suggested!

**Comment by meghalithic** [comment]
@johnbradley how does it look now?

**Review by johnbradley — APPROVED** [review]
LGTM

**Pull request merged**